.circle/deployment

```diff
@@ -21,6 +21,20 @@ fi
 # TODO: remove the install_gh call once circleci config is updated in all packs
 type gh &>/dev/null || ~/ci/.circle/install_gh
 
+# GH_TOKEN is used by gh which is used in functions.sh
+export GH_TOKEN=${MACHINE_PASSWORD}
+source ~/ci/tools/functions.sh
+
+DEFAULT_BRANCH=$(_gh_default_branch)
+# Give a good error message after the first use of github API in this script.
+if $? != 0; then
+    echo "Error retrieving data with github graphql API."
+    echo "The GitHub PAT might need to be regenerated:"
+    echo "https://github.com/settings/tokens/new?scopes=public_repo&description=CircleCI%3A%20stackstorm-${PACK_NAME}"
+else
+    echo "GitHub PAT is active."
+fi
```

Comment:
It is also possible to use `git` to query for the default branch: https://stackoverflow.com/a/44750379/1134951

But if we use `git` to retrieve the default branch, we still need an API call at this point in the script to validate that the PAT is still active.

Each API call counts towards our API usage limit. So, I would prefer that we use the return value of the API call for something productive; getting the default branch is productive.